File: Programing_Basics_with_JS/conditionalStatementsMoreExercises/05.Pets.js
function pets(input) {
    let numbersOfDays = Number(input[0]);
    let foodInKilograms = Number(input[1]);
    let dogFoodPerDayKilograms = Number(input[2]);
    let catFoodPerDayKilograms = Number(input[3]);
    let turtleFoodPerDayGrams = Number(input[4]);

    let turtleFoodPerDayKilograms = turtleFoodPerDayGrams / 1000;
    let allEatenFoodPerDay = dogFoodPerDayKilograms + catFoodPerDayKilograms + turtleFoodPerDayKilograms;
    let allEatenFood = allEatenFoodPerDay * numbersOfDays;
    let foodDiff = Math.abs(allEatenFood - foodInKilograms);
    
    if (foodInKilograms >= allEatenFood) {
        console.log(`${Math.floor(foodDiff)} kilos of food left.`);
    } else {
        console.log(`${Math.ceil(foodDiff)} more kilos of food are needed.`);
    }
}

pets(['2', '10', '1', '1', '1200'])
pets(['5', '10', '2.1', '0.8', '321'])

File: Programming_Basics_with_Python/conditional_statements_more_exercise/05._pets.py
from math import floor, ceil


def pets_food(days, food_in_kg, dog_food_per_dey_kg, cat_food_per_dey_kg, turtle_food_per_dey_gr):
    turtle_food_per_dey_kg = turtle_food_per_dey_gr / 1000
    pets_needed_food_per_day = dog_food_per_dey_kg + cat_food_per_dey_kg + turtle_food_per_dey_kg
    pets_needed_food_for_all_days = days * pets_needed_food_per_day
    food_difference = food_in_kg - pets_needed_food_for_all_days

    if food_difference >= 0:
        return f'{floor(food_difference)} kilos of food left.'
    else:
        return f'{ceil(abs(food_difference))} more kilos of food are needed.'


days = int(input())
food_in_kg = int(input())
dog_food_per_dey_kg = float(input())
cat_food_per_dey_kg = float(input())
turtle_food_per_dey_gr = float(input())

print(pets_food(days, food_in_kg, dog_food_per_dey_kg, cat_food_per_dey_kg, turtle_food_per_dey_gr))